movement/movement.cpp

```diff
@@ -1605,18 +1602,18 @@ int cmdLock(const std::shared_ptr<Player>& player, cmd* cmnd) {
         return(0);
     }
 
-    if(player->flagIsSet(P_SITTING)) {
-        player->print("You have to stand up first.\n");
-        return(0);
-    }
-
+    
     exit = findExit(player, cmnd);
 
     if(!exit) {
         player->print("Lock what?\n");
         return(0);
     }
 
+    if(player->flagIsSet(P_SITTING))
```

Comment:
nit: since this is everywhere, probably could have done a player->checkStand() function; but doesn't really matter.